### PR TITLE
DE-Translation: Revised strings 1001 to 1821

### PR DESCRIPTION
**Pull request by violine1101**
First part was pull request #394.

### CODE REVIEW

File: lang/de_DE.trn [hunk 1]
```diff
@@ -1116,11 +1116,11 @@ t525 Z:
 o526 f: 
 t526 f:
 o527 Seed: 
-t527 Seed:
+t527 tartwert:
```
Comment (L3ice) [inline lang/de_DE.trn]: :scream:

File: lang/de_DE.trn [hunk 2]
```diff
@@ -1748,25 +1748,25 @@ t807 Welttyp:
...
-t812 Erleuchtet Chunks
+t812 Beleuchtet Chunks
 o813 This world was not downloaded from a FTP server. Uploading worlds that were not downloaded from a FTP server is currently not possible
-t813 Diese Welt wurde nicht von einem FTP Server heruntergeladen. Hochladen von Welten, die nicht von einem FTP Server heruntergeladen wurden, ist momentan noch nicht möglich.
+t813 Diese Welt wurde nicht von einem FTP-Server heruntergeladen. Hochladen von Welten, die nicht von einem FTP-Server heruntergeladen wurden, ist momentan noch nicht möglich.
 o814 Window size increased. You may have problems using the cursor until MCEdit is restarted.
 t814 Fenster wurde vergrössert. Du hast vielleicht Probleme beim Benutzen des Mauszeigers, bis MCEdit neugestartet wurde.
 o815 FTP Server IP:
-t815 FTP Server IP:
+t815 FTP#Server IP:
```
Comment (L3ice) [inline lang/de_DE.trn]: :no_mouth:

File: lang/de_DE.trn (hunk 1 above)
Comment (violine1101) [inline lang/de_DE.trn]: Fixed 😊

File: lang/de_DE.trn (hunk 2 above)
Comment (violine1101) [inline lang/de_DE.trn]: Fixed 😊

**Pull request merged**